README.md

```diff
@@ -17,14 +17,20 @@ This library is used directly by applications only.
 
 This library can be used in the **browser** only.
 
+## Note
+
+This plugin is now registered with npm and jspm as `aurelia-notification`. This simplifies the installation.
+
 ## Installation
 
 Installing this module is fairly simple.
 
-Run `jspm install github:spoonx/aurelia-notification` from your project root.
+Run `jspm install aurelia-notification` from your project root.
 
 Either set your own styles or install humane.js with `jspm install github:wavded/humane-js` to have some default styles to choose from.
 
+Npm installation for eg webpack is currently not possible.
```

Comment:
Isn't it? I thought this PR would fix that. Surely it's possible if you bundle the deps?